sdk/healthinsights/health-insights-radiologyinsights-rest/review/health-insights-radiologyinsights.api.md

```diff
@@ -0,0 +1,1090 @@
+## API Report File for "@azure-rest/health-insights-radiologyinsights"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { Client } from '@azure-rest/core-client';
+import { ClientOptions } from '@azure-rest/core-client';
+import { CreateHttpPollerOptions } from '@azure/core-lro';
+import { ErrorModel } from '@azure-rest/core-client';
+import { ErrorResponse } from '@azure-rest/core-client';
+import { HttpResponse } from '@azure-rest/core-client';
+import { KeyCredential } from '@azure/core-auth';
+import { OperationState } from '@azure/core-lro';
+import { RawHttpHeaders } from '@azure/core-rest-pipeline';
+import { RawHttpHeadersInput } from '@azure/core-rest-pipeline';
+import { RequestParameters } from '@azure-rest/core-client';
+import { SimplePollerLike } from '@azure/core-lro';
+import { StreamableMethod } from '@azure-rest/core-client';
+
+// @public
+export interface AgeMismatchInferenceOutput extends RadiologyInsightsInferenceOutputParent {
+    kind: "ageMismatch";
+}
+
+// @public
+export interface Annotation extends Element {
+    authorString?: string;
+    text: string;
+    time?: string;
+}
+
+// @public
+export interface AnnotationOutput extends ElementOutput {
```

Comment:
Wondering if `type AnnotationOutput = ElementOutput & Annotation` might read better and prevent redeclaring properties?